messages/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+
+	//"dont-slack-evil/apphome"
+)
+
+// Response is of type APIGatewayProxyResponse since we're leveraging the
+// AWS Lambda Proxy Request functionality (default behavior)
+//
+// https://serverless.com/framework/docs/providers/aws/events/apigateway/#lambda-proxy-integration
+type Response events.APIGatewayProxyResponse
+
+var slackOauthToken = os.Getenv("SLACK_OAUTH_TOKEN")
+var api = slack.New(slackOauthToken)
+
+// Handler is our lambda handler invoked by the `lambda.Start` function call
+func Handler(ctx context.Context, body []byte) (Response, error) {
+	buf := new(bytes.Buffer)
+	resp := Response{
+		IsBase64Encoded: false,
+		Headers: map[string]string{
+			"Content-Type": "application/json",
+		},
+	}
+	eventsAPIEvent, e := slackevents.ParseEvent(json.RawMessage(body), slackevents.OptionVerifyToken(&slackevents.TokenComparator{VerificationToken: slackOauthToken}))
```

Comment:
Hey, are you sure the VerificationToken is indeed the OAuth token @mourtisma ? I remember that they were 2 distinct tokens, and btw the verification token is deprecated in favor of the "signin secret" which behaves almost exactly like a verificationToken (at least in the node sdk/bolt framework for slack)

The verification token / signin secret is available here : https://api.slack.com/apps/AU7RW3DLG/general